core/kubernetes/main/mongo/mongo-service.yaml

```diff
@@ -1,12 +1,13 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: openhim-mongo-service
+  name: mongo
```

Comment:
Lets keep the naming of the service with a postfix of `-service` so that its the same as the other services. If this is redundant then we should remove it from the other as well, but maybe lets keep it for now and can update at a later stage

Reply:
addressed https://github.com/openhie/instant/pull/44/commits/461f957fada1c5f1e1a1d02cc9281189903b128b